test_ffi/tests/test.js

```diff
@@ -221,7 +217,11 @@ if (!(status & (1 << 4))) {
 dylib.symbols.print_buffer(ptr0, 8);
 const ptrView = new Deno.UnsafePointerView(ptr0);
 const into = new Uint8Array(6);
+// Make sure the buffer does not get collected
+globalThis.into = into;
```

Comment:
This seems strange, can you elaborate what's the reason for that?

Reply:
I think I was trying to get rid of the flake in FFI tests, thinking that it might've been about V8 garbage collecting a buffer while a pointer number from `Deno.UnsafePointer.of(into)` is still being used.

However, looking at this now it's not definitely necessary. I'll remove these changes.

Reply:
I'll leave the `.pointer` changes as the `ptr()` function doesn't really serve much of a purpose.